### ISSUE TITLE
Frame Configuration fails

### ISSUE THREAD

**Issue by testtesttesttesttsetset** [Xian55/WowClassicGrindBot#624]
### Description

Have the below logs:
08:21:42 | DataFrames 0 - Texture: Size [ Width=1, Height=1 ]
08:21:42 | Frame configuration was incomplete! Please try again, after resolving the previusly mentioned issues...
08:21:42 | Addon Rect(Size [ Width=0, Height=0 ]) size issue. Either too small or too big!
08:21:42 | DataFrameMeta { Hash = 16645925, Spacing = 1, Sizes = 66, Rows = 45, Count = 925 }
08:21:42 | [Enter] press random 118ms
08:21:42 | [Enter] press random 111ms
08:21:41 | /hunter
08:21:41 | Enter configuration mode.
08:21:41 | Addon installed! Version: 1.7.60
08:21:41 | Client window: Rectangle [ X=664, Y=181, Width=1920, Height=1080 ]
08:21:41 | Found WowProcess with pid=14484 WowClassic

Here are pics of the addon in config mode and normal mode:
https://imgur.com/TRdnh9R
https://imgur.com/j2Szys2

Here is a pic of the results for auto config
https://imgur.com/gN6c9tu

Manual config shows similar error and is of no help

### Addon Version

1.7.60

### World of Warcraft Client

World of Warcraft Classic

### Reproduction Steps

Follow readme instructions (all config is as expected, 1920x1080 windowed mode, all brightness ets is default)

### Screenshots

![image](https://github.com/user-attachments/assets/5e22ab1d-9330-4a5a-a187-2f5e441e7367)
![j2Szys2](https://github.com/user-attachments/assets/31b1258f-cd27-4d47-8de7-3b788d982f58)
![TRdnh9R](https://github.com/user-attachments/assets/40ba818c-6e22-4eaf-96b9-3a07e5598db0)

### Logs

<html><body>

08:21:42 | ResetConfigState
-- | --
08:21:42 | DataFrames 0 - Texture: Size [ Width=1, Height=1 ]
08:21:42 | Frame configuration was incomplete! Please try again, after resolving the previusly mentioned issues...
08:21:42 | Addon Rect(Size [ Width=0, Height=0 ]) size issue. Either too small or too big!
08:21:42 | DataFrameMeta { Hash = 16645925, Spacing = 1, Sizes = 66, Rows = 45, Count = 925 }
08:21:42 | [Enter] press random 118ms
08:21:42 | [Enter] press random 111ms
08:21:41 | /hunter
08:21:41 | Enter configuration mode.
08:21:41 | Addon installed! Version: 1.7.60
08:21:41 | Client window: Rectangle [ X=664, Y=181, Width=1920, Height=1080 ]
08:21:41 | Found WowProcess with pid=14484 WowClassic
08:21:41 | ResetConfigState
08:21:41 | DataFrames 0 - Texture: Size [ Width=1, Height=1 ]
08:19:13 | ResetConfigState
08:19:13 | DataFrames 0 - Texture: Size [ Width=1, Height=1 ]
08:19:13 | Frame configuration was incomplete! Please try again, after resolving the previusly mentioned issues...

</body>
</html>

**Comment by Xian55** [comment]
Gooday can you tell me which version of the game is that
* WoW 20th Anniversary ?

**Comment by Xian55** [comment]
Can you run the following command in game

```lua
/dump WOW_PROJECT_ID
```

and can you send me the result ?

**Comment by Xian55** [comment]
I can recommend to follow this setup: 
* https://github.com/Xian55/WowClassicGrindBot/wiki/Guidance-for-good-DataFrame#optional-new-way---increase-addon-cell-size
* https://github.com/Xian55/WowClassicGrindBot/pull/402

After trying these options and the issue still persist, let me know!

**Comment by testtesttesttesttsetset** [comment]
WOW_PROJECT_ID=2

Yes, classic anniversary edition 

I've tried with cell size 1,2,3 and the issue remains there. Ive tried different (supported) resolutions, and every time the error is the same, however the 'DataFrameMeta' values differ. 

I am running it on a dark static background (as can be seen in the pics)

**Comment by Xian55** [comment]
Alright thanks for letting me know this info. Later on today i will try to look into what's going on.

Meanwhile please disable all other addons what you might have. There are two addons what the project is using
* DataToColor / your custom addon name
* Bindpad

Can you start the game without any other addons enabled beside these two ?

There are other addons which might tinker with the default GuiScale.

**Comment by testtesttesttesttsetset** [comment]
Yes, I've already disabled all other addons besides this and the error persists. I will continue to mess with the configuration

**Comment by Xian55** [comment]
I can recommend changing the resolution and trying out fullscreen and windowed modes.

Note: I have not personally tested whether the project works with the anniversary edition.

**Comment by testtesttesttesttsetset** [comment]
yes I've already done so, I've tried every resolution and different modes. The values change depending on the SIZE variable, but error remains
for ex.: <html><body>

09:01:49 | DataFrameMeta { Hash = 16732671, Spacing = 1, Sizes = 67, Rows = 32, Count = 671 }
-- | --



</body>
</html>

**Comment by necro12345** [comment]
Bumping as I am having the same issue. I would like to point out that when the addon runs it's in-game command, the red dot does turn red. However, I went and downloaded the DataToColor addon from HappyPixels and when I run /datatocolors the dot is now green. I was going to rummage around your code to see where I could change the command line of the blaze server but, I think you would be faster.

Quick Edit:

I tried to be cheeky by creating the addon and naming it datatocolor then replace the addon files from the datatocolor files, while it does cycle and go to a green dot, it still does not detect frames.

**Comment by necro12345** [comment]
Just as an update - I was able to get this working by unplugging on of my monitors? I had 3, 2+TV, when I unplugged on and moved it to another screen, it started working. Though a bit clunky and IDK if it's a frame issue or not, when I load, say, the mage profile, he just jumps in place and looks around. Little goofy, but I'm sure I just need to play with it.

**Comment by Xian55** [comment]
> Just as an update - I was able to get this working by unplugging on of my monitors? I had 3, 2+TV, when I unplugged on and moved it to another screen, it started working. Though a bit clunky and IDK if it's a frame issue or not, when I load, say, the mage profile, he just jumps in place and looks around. Little goofy, but I'm sure I just need to play with it.

I've heard this before, a guy mentioned that after unplugging a monitor suddenly resolves the problem. 

I myself use 3 monitor setup, i personally not encountered this kind of issue so if you have a few minutes to describe your setup i might be able to find out what's going on.

* Are you using Anniversary version ?
* How many monitors are you using ?
* What resolutions and refresh rate are the monitors have ?
* On which monitor you use WoW client ?
* Does that monitor the primary monitor ?
* Can you Run `dxdiag.exe` on your machine and upload the result  here ?
* After you start the World of Warcraft client do you move the client to another monitor  with (Win+Arrrow Keys) ?

**Comment by Xian55** [comment]
Today i had time to check the project, and its working with the **Anniversary** version, and i can confirm that my setup works as the prior versions there are no changes in the Wow client. Also there are no LUA errors.

The `Bindpad` addon shows warnings but can be ignored for now.

I've tried my setup with 
* Opening wow client in the Primary Monitor
* Windowed mode - 1024x768 resolution
* BlazorServer with `Auto` and `Manual` Frame config setup
* HeadlessServer works fine after the frameconfig.

When you see the following image below that means the BlazorServer was unable to read any kind of Info.
The `Red dot` text with green outline means nothing and it just misleading.
![image](https://github.com/user-attachments/assets/3e86ae81-f477-4131-a76e-8813709ca862)

**Comment by Xian55** [comment]
> yes I've already done so, I've tried every resolution and different modes. The values change depending on the SIZE variable, but error remains for ex.:
> 
> 09:01:49	DataFrameMeta { Hash = 16732671, Spacing = 1, Sizes = 67, Rows = 32, Count = 671 }

That does not seems right at all.

The following should be seen:
* DataFrameMeta { Hash = xxxxxxx, Spacing = 1, Sizes = 1-[Config.CellSize], Rows = 1, Count = 102 }

Can you tell me more about your setup
* Are you using Anniversary version ?
* How many monitors are you using ?
* What resolutions and refresh rate are the monitors have ?
* On which monitor you use WoW client ?
* Does that monitor the primary monitor ?
* Can you Run `dxdiag.exe` on your machine and upload the result here ?
* After you start the World of Warcraft client do you move the client to another monitor with (Win+Arrrow Keys) ?

**Comment by Xian55** [comment]
Just a quick heads up, the game client window has to be **always visible in order to complete the frame configuration** process.

If you see the WoW game client window on the screen, then the BlazorServer going to see it as well.

If you have to switch between windows / Alt-Tab be sure to resize the browser window in order to don't obstruct the view of the top left corner of the game client window.

---

If you are running the World of Warcraft client with Administrator mode be sure to avoid that and or run the BlazorServer as higher level privileges as the WoW Client.

If the WoW client is ran with Admin rights and the `BlazorServer` is not, then the `BlazorServer` wont be able to send keystrokes to the WoW client.

The best approach would be to do not run any of the applications in `Administrator mode`

**Comment by Xian55** [comment]
Closing due no response from the issue opener whether the issue has been resolved.
